### ISSUE TITLE
4.0.2: Grails console startup fails with MissingMethodException, JMenu.setMnemonic

### ISSUE THREAD

**Issue by longwa** [grails/grails-core#11510]
Starting with Grails 4.0.2, creating a new project and then running `grails console` results in a startup error:
```
2020-03-27 15:03:46.361 ERROR --- [  restartedMain] o.s.boot.SpringApplication               : Application run failed

groovy.lang.MissingMethodException: No signature of method: javax.swing.JMenu.setMnemonic() is applicable for argument types: (String) values: [F]
Possible solutions: setMnemonic(char), setMnemonic(int), getMnemonic()
        at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:70)
        at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:76)
        at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.callCurrent(PogoMetaClassSite.java:76)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:156)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:176)
        at groovy.ui.view.MacOSXMenuBar$_run_closure1.doCall(MacOSXMenuBar.groovy:58)
        at groovy.ui.view.MacOSXMenuBar$_run_closure1.doCall(MacOSXMenuBar.groovy)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```

Looks like it might be due to this issue:
https://issues.apache.org/jira/browse/GROOVY-8253

It does work in 4.0.1.

Running with OpenJDK 1.8.0_242, macOS.

**Comment by pinei** [comment]
Same problem with grails-4.0.4.

The groovy issue wasn't resolved in 2017?

**Comment by puneetbehl** [comment]
Please update the Groovy version to `2.6.0-alpha-1` or higher. 

```
configurations.all {
    resolutionStrategy.eachDependency { DependencyResolveDetails details ->
        if(details.requested.group == 'org.codehaus.groovy') {
            details.useVersion('2.6.0-alpha-4')
        }
    }
}

```

**Comment by puneetbehl** [comment]
This issue is fixed with Grails 4.1.0.M2
